### PR TITLE
Adds framework for alternative (and gender sensitive) job outfits & provided items. Adds scrubs, EMT, and nurse outfits for doctors.

### PR DESCRIPTION
**Pull request by ShizCalev**
:cl: ShizCalev
add: Added the ability to create alternative outfits for jobs. These outfits have a random chance of being equipped, and can be configured by the server (or if someone has a really cool outfit they can force a higher probability by coding it as such.) These outfits can also be gender specific.
add: Added the ability for different items/equipment to be given to mobs when they spawn, and for them to have probabilities of spawning in general (ie you can now make it so an assistant spawns with a crowbar or an esword in their hands, or nothing at all!)
add: MD's now have a chance to spawn in dressed up in scrubs, surgeon outfits, EMT clothing, or for females, nurse outfits!
add: Medical staff members now have a chance of spawning in wearing latex gloves.
/:cl:

Seemed like a good idea to help diversify the station a bit more since having 7 doctors all wearing the same exact thing when they start felt a little bland.

Plenty of opportunity to expand on top of it.

Enabling it is a config option, as is tweaking the probability of someone being dressed an alternative outfit.

### CODE REVIEW

**Comment by nicbn** [comment]
The points of uniforms is that they are... Uniform.

**Comment by Vidiveni** [comment]
This way it will be less fun to dress up, since other people will randomly dress up without any effort.

**Comment by DaxDupont** [comment]
Is it a server or client option?

It would probably have both. So both people and server can opt out. I'd hate to change all my clothes on roundstart as a medic.

**Comment by ShizCalev** [comment]
@DaxDupont It's both.

**Comment by Iamgoofball** [comment]
I had a system I made awhile back when people were fighting the security uniform wars that would dynamically change the used icon for every item of the type, lemme see if I can find it

**Comment by Iamgoofball** [comment]
>scrubs, surgeon outfits, EMT clothing, or for females, nurse outfits!

No need to gender lock any outfits, it's sexist.

File: code/modules/clothing/suits/armor.dm [hunk 1]
```diff
@@ -56,8 +56,6 @@
 	desc = "A trenchcoat enhanced with a special lightweight kevlar. The epitome of tactical plainclothes."
 	icon_state = "hostrench"
 	item_state = "hostrench"
-	flags_inv = 0
-	strip_delay = 80
```
Comment (ShizCalev) [inline code/modules/clothing/suits/armor.dm]: duplicate vars.

**Comment by Steelpoint** [comment]
I'd prefer the HoS keep their current round start uniform choice instead of making it a 50/50 affair.

**Comment by PKPenguin321** [comment]
could make it a pref like backpacks

**Comment by ShizCalev** [comment]
It is a pref. Look up lmao

**Comment by BeeSting12** [comment]
Not a fan of the HoS one. If they want to wear their great coat they can put it on from the locker. The trench coat doesnt spawn on some maps IIRC.

**Comment by epochayur** [comment]
the first thing I do any time I play MD is go get into the EMT outfit, so +1

**Comment by fludd12** [comment]
I kinda wish it were possible to, instead of having the preference be for alternate outfits in general, to be more a preference like how hairstyles were done - you choose *which* alternate outfit you want.

...Of course, that'd require some potentially messy changes to the job selection menu.

**Comment by Borisvanmemes** [comment]
Add a male nurse outfit too and this'll be good stuff.

**Comment by Mickyan** [comment]
Probably out of scope but here's an idea, instead of being a yes/no preference to random alternate outfits, allow choosing a preference of outfit style from a list (ex. "regular", "alternate", "casual", "formal") that is applied across all your jobs, similar to the selection of departmental backpacks/satchels

If you pick a style that isn't defined for the job you spawn as, it defaults to the regular uniform

**Comment by miniusAreas** [comment]
@ShizCalev does this mean we will see more jumpskirts other than the security ones?

**Comment by ShizCalev** [comment]
Should finish it up in the next few days.

**Comment by anconfuzedrock** [comment]
This makes me wanna add hijabs

**Comment by ShizCalev** [comment]
Will finish this up when I get back from vacation.

**Comment by Nabski89** [comment]
Hey, I'm working on a bunch of departmental skirts.

I also hate github.

If I gave you the dmi for it would you be willing to add those in with this?

**Pull request closed without merging**